### DISCUSSION TITLE
Changing the GUI Builder (Design) theme.

### DISCUSSION THREAD

**Discussion by SnowySunnyDaze** [apache/netbeans#6957] [General]
Hello everyone.
I'm currently using FlatLaf Dark for the Look and Feel of the IDE but I want to keep the background color of the JFrame and JPanel generated by the GUI Builder as white instead of black.
Is there any way to change the GUI Builder to light mode while keeping the IDE's appearance as dark mode?

**Comment by Chris2011** [comment]
Maybe @DevCharly can tell here smth. Maybe there are properties just for the gui builder. But I dunno, unfortunately.

**Comment by mbien** [comment]
you can preview the component in different LAFs, but you can't change the form editor LAF specifically, it is part of NB and uses the same LAF.
![image](https://github.com/apache/netbeans/assets/114367/86929d48-63e1-46b9-ab6d-9cc700d9ad21)
